src/components/Facepile/Facepile.tsx

```diff
@@ -57,14 +59,13 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   }
```

Comment:
The _calculateNumPersonasTo show method is unnecessary.

1. maxDisplayablePersonas will *default* to the value from defaultProps, you don't need to reference it like this.
2. in the renderMethod you can have:

```ts
let { personas, max } = this.props;
let visibleCount = Math.min(personas.length, max);
let overflowCount = personas.length - visibleCount;

return (
   ...stuff...
   { (overflowCount > 0) && (
   { this._getOverflowElement() }
   ) }
);
```